runelite-client/src/main/java/net/runelite/client/plugins/mining/MiningOverlay.java

```diff
@@ -108,4 +118,16 @@ public Dimension render(Graphics2D graphics)
 		}
 		return null;
 	}
+
+	/**
+	 * Checks if the given point is "upstairs" in the mlm.
+	 * The upper floor is actually on z=0.
+	 *
+	 * @param localPoint
+	 * @return true if localPoint is at same height as mlm upper floor
+	 */
+	boolean isUpstairsMotherlode(LocalPoint localPoint)
```

Comment:
We already have a similar function available in the MLM plugin. Seems like it might be worthwhile to move it to a common location and reference it from both plugins?

Reply:
Is this what you mean? https://github.com/runelite/runelite/compare/master...Twiglet1022:mlm-same-level-timers-2